app/lib/task/handlers.dart

```diff
@@ -1,29 +1,108 @@
+import 'dart:convert';
 import 'dart:io' show gzip;
 
-import 'package:mime/mime.dart';
+import 'package:pub_dev/dartdoc/dartdoc_page.dart';
+import 'package:pub_dev/package/backend.dart';
 import 'package:pub_dev/shared/exceptions.dart';
 import 'package:pub_dev/shared/handlers.dart';
+import 'package:pub_dev/shared/redis_cache.dart';
 import 'package:pub_dev/task/backend.dart';
 import 'package:shelf/shelf.dart' as shelf;
 
+const _safeMimeTypes = {
+  // Binary image files are generally safe to serve
+  'avif': 'image/avif',
```

Comment:
I think it is annoying in `package:mime` that this extension-> expected content type mapping is not exposed (IIRC). Let's file an issue to expose these and also a TODO here to use it instead?

Reply:
You can just lookup the extension, but I intentionally want to limit this to safe list of safe-to-serve mimetypes.

Serving SVG is not safe because it can embed scripts..